jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/CloseStatus.java

```diff
@@ -175,16 +175,11 @@ public static CloseStatus getCloseStatus(Frame frame)
     // TODO consider defining a precedence for every CloseStatus, and change SessionState only if higher precedence
     public static boolean isOrdinary(CloseStatus closeStatus)
     {
-        switch (closeStatus.getCode())
-        {
-            case NORMAL:
-            case SHUTDOWN:
-            case NO_CODE:
-                return true;
-
-            default:
-                return false;
-        }
+        int code = closeStatus.getCode();
+        if (code == NORMAL || code == NO_CODE || code >= 3000)
```

Comment:
I kind of prefer switches for this kind of operation....  I suspect they are more efficient also?

Reply:
just thought id go for simplicity, this code could just be 
`return (code == NORMAL || code == NO_CODE || code >= 3000);`

instead of something like this
```
switch (code)
{
    case NORMAL:
    case NO_CODE:
        return true;
                
    default:
        if (code >= 3000)
            return true;
        return false;
}
```

Reply:
OK, I can live with that... I guess the NORMAL close is the most frequent so it will be executed first.